docs/deepsemgrep.md

```diff
@@ -1,37 +1,61 @@
 ---
 slug: deepsemgrep
+append_help_link: true
 description: "This document provides an overview of DeepSemgrep use cases, such as its use in type inferences, class inheritance, constant propagation, taint analysis."
 ---
 
+import MoreHelp from "/src/components/MoreHelp"
+
 # DeepSemgrep
 
 ## Introduction
 
 Improve your scan results for entire codebases with interfile coding paradigms using DeepSemgrep instead of Semgrep's regular intrafile (within-a-single-file) approach. DeepSemgrep empowers you to easily scan whole repositories that have object-oriented programming paradigms with classes in different files to find vulnerabilities in your code. DeepSemgrep is a proprietary extension of free and open source Semgrep which leverages global analysis tools, and uses the same rules as Semgrep. 
 
-This document demonstrates the utility of DeepSemgrep through use cases, guiding you through examples of Semgrep rules. Both Semgrep rules and code on which rules are tested are marked as code snippets and introduced always as a rule, code, or file used to illustrate the capabilities of DeepSemgrep.
+This document demonstrates the utility of DeepSemgrep through use cases, guiding you through examples of Semgrep rules. Both Semgrep rules and code on which rules are tested are marked as code snippets and introduced always, as a rule, code, or file used to illustrate the capabilities of DeepSemgrep.
+
+### DeepSemgrep language support
+
+DeepSemgrep now offers alpha support for Java, JavaScript, and TypeScript.
 
-### Obtaining DeepSemgrep
+## Installing DeepSemgrep
 
-To get and install DeepSemgrep, follow these steps:
+To enable DeepSemgrep installation, schedule a product demo by following these steps:
 
 1. Submit your email using the [DeepSemgrep beta form](https://semgrep.dev/deep-semgrep-beta).
-2. Follow the steps and instructions in the email you receive from the Semgrep team.
-3. Start using DeepSemgrep with the following command:
+1. Follow the steps and instructions in the email you receive from the Semgrep team and schedule a product demo.
+
+After your product demo, you'll get access to a new ruleset! Follow the instructions below to get your findings.
+
+:::info Prerequisite
+- Local installation of Semgrep CLI. See [Getting started with Semgrep CLI](/getting-started) to install Semgrep CLI.
+:::
+
+To install DeepSemgrep, follow these steps:
+
+1. Log in to Semgrep CLI with the following command:
     ```sh
     semgrep login
     ```
-4. Follow the link that Semgrep prints to the command-line.
-5. To install DeepSemgrep, use the following command:
+1. Follow the link that Semgrep printed in the command-line.
+1. To install DeepSemgrep, use the following command:
     ```sh
     semgrep install-deep-semgrep
     ```
-6. To test DeepSemgrep, use the following command with your own rules:
-    ```sh
-    semgrep --deep --config=myrule.yaml
+1. To test DeepSemgrep, use the following command:
```

Comment:
```suggestion
1. To test DeepSemgrep, use the following command in the root directory of the codebase to scan:
```
We don't tell people where (what directory) to run the command in.

Reply:
@s-santillan great idea!